src/read.c

```diff
@@ -4981,13 +4981,13 @@ static avifResult avifDecoderDecodeTiles(avifDecoder * decoder, uint32_t nextIma
 
         // Scale the decoded image so that it corresponds to this tile's output dimensions
         if ((tile->width != tile->image->width) || (tile->height != tile->image->height)) {
-            if (!avifImageScale(tile->image,
-                                tile->width,
-                                tile->height,
-                                decoder->imageSizeLimit,
-                                decoder->imageDimensionLimit,
-                                &decoder->diag)) {
-                avifDiagnosticsPrintf(&decoder->diag, "avifImageScale() failed");
+            if (!avifImageScaleWithLimit(tile->image,
+                                         tile->width,
+                                         tile->height,
+                                         decoder->imageSizeLimit,
+                                         decoder->imageDimensionLimit,
+                                         &decoder->diag)) {
+                avifDiagnosticsPrintf(&decoder->diag, "avifImageScaleWithLimit() failed");
```

Comment:
I think we should delete this `avifDiagnosticsPrintf()` call because it overwrites the better diagnostic message written by `avifImageScaleWithLimit()`. (You can do this in a separate pull request because this is a bug fix.)

Reply:
Acknowledged.

Reply:
Done in #1666.